app/models/miq_server/environment_management.rb

```diff
@@ -29,7 +29,7 @@ def minimal_env_options
         options.shift # remove the "minimal" from the front of the array
 
         # Special case where Netbeans is handling the UI worker for debugging
-        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting noui) : o }.flatten
+        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting no_ui) : o }.flatten
```

Comment:
Why is this netbeans/debugger option even here? Why do we assume an IDE would would a schedule/reporting worker? Wouldn't that just be another case of `no_ui`? ("I'm a netbeans user, I should use `MIQ_SPARTAN=minimal:no_ui:schedule:reporting` want netbeans to work and I want those two workers")